### ISSUE TITLE
OS-X Client crash - seems related to a specific folder tree...

### ISSUE THREAD

**Issue by paulrbarnard** [owncloud/client#7519]
### Expected behaviour
Client should not crash

### Actual behaviour
Client always crashes while syncing a specific folder tree.

### Steps to reproduce
1. Launch ownCloud client 
2. Wait for it to sync the problem folder tree
3. It always crashes 100% repeatable - not this only happens on one computer my other computer syncs without issue with the same client and OS

### Server configuration
Operating system: Univention 4.4-2 errata294

Web server: 

Database: 

PHP version:

ownCloud version: 10.2.1-1

Storage backend (external storage):

### Client configuration
Client version:  Version 2.5.4 (build 11456).

Operating system:  OS-X 10.14.6 (18G95)

OS language: English

Qt version used by client package (Linux only, see also Settings dialog):

Client package (From ownCloud or distro) (Linux only):

Installation path of client: Default Applications folder

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

```Template for output < 10 lines```

1. Client logfile: Output of `owncloud --logwindow` or `owncloud --logfile log.txt`
Unable to get any log related to the crash.  The command doesn't run once the app has crashed and running before the crash just shows normal operation.
I have reported the crash from the crash reporter many times
(On Windows using `cmd.exe`, you might need to first `cd` into the ownCloud directory)
(See also http://doc.owncloud.org/desktop/2.2/troubleshooting.html#client-logfile )

2. Web server error log:

3. Server logfile: ownCloud log (data/owncloud.log):

**Comment by michaelstingl** [comment]
@paulrbarnard Did you sent a crash report? Could you post the ID? When you reproduce, please capture also the client logfile and post the last few lines…

**Comment by ogoffart** [comment]
> Wait for it to sync the problem folder tree

What is there special in this folder tree?
There must be something unusual because most people can sync without problems.

Maybe you can also try the 2.6 rc.

**Comment by paulrbarnard** [comment]
@michaelstingl I'll post the next one as i don't have the ID for the ones already sent
@ogoffart The specific folder tree is my development path.  I'm trying to narrow it down by enabling sub folders to sync one at a time.  There shouldn't be anything odd in there and one computer is handling it fine, my laptop crashes consistently.

**Comment by paulrbarnard** [comment]
Good news and bad news.
To attempt to narrow down which specific folder was causing the problem I unchecked all the folders in the folder tree on my laptop.  This removed them all from synchronising and I then added them back in one by one.
The good news is all the folders are now synchronising on my laptop without crashing.
The bad news is that it points to the problem being something specific to the files as they were on  the laptop and the removal of sync and re-enable of sync has cleared the problem.  I will make sure to capture logs if this occurs again but right now the problem is not reproducible.
